src/Microsoft.NetCore.Analyzers/UnitTests/Security/UseRSAWithSufficientKeySizeTests.cs

```diff
@@ -285,6 +269,21 @@ public void TestMethod()
             GetCSharpResultAt(9, 28, UseRSAWithSufficientKeySize.Rule, "system.security.cryptography.asymmetricalgorithm"));
         }
 
+        [Fact]
+        public void TestReturnObjectOfRSADerivedClassNoDiagnostic()
```

Comment:
Can you also add a test case from the example provided in the bug?

Reply:
+1